organization/_posts/2017-10-12-startup.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "HSF Weekly Meeting #112, October 12, 2017"
+layout: default
+---
+
+# HSF Weekly Meeting #112, October 12, 2017
+
+#### *Present*: Graeme Stewart, Benedikt Hegner, Dario Menasce, Liz Sexton-Kennedy, Ian Bird, Rob Kutschke, Michel Jouvin, Guilherme Amadio, Pere Mato, Mike Sokoloff, Pete Elmer, David Lange, John Harvey, Daniel Elvira, Marko Petric
+
+## General News
+
+### WLCG/HSF Workshop
+
+This has changed from being 2 parallel workshops to a single workshop, with both parallel and plenary sessions for the two communities. [Draft announcement prepared](https://docs.google.com/document/d/18dDKX11gWxDCEb9lagWj9BSlMGAUUHLYUOBuSxYroMo/edit?usp=sharing). Workshop will run from Monday 26 March at 9am until Thursday 29 March at noon.
+
+A straw man agenda:
+- Monday morning: common welcome session with WLCG/HSF
+- Monday morning/afternoon: short presentations by working groups
```

Comment:
You are lacking a blank line here, thus the bullet list is interpreted as part of the § "a straw man....". Not sure why pandoc didn't make it right...